test/e2e/cloud.e2e.js

```diff
@@ -246,5 +277,97 @@ describe('Cloud Commands [@device]', () => {
 		expect(stderr).to.equal('');
 		expect(exitCode).to.equal(0);
 	});
+
+	it('Fails to name an unknown device', async () => {
+		const invalidDeviceID = '1234567890';
+		const args = ['cloud', 'name', invalidDeviceID, 'NOPE'];
+		const { stdout, stderr, exitCode } = await cli.run(args);
+		const log = [
+			`Renaming device ${invalidDeviceID}`,
+			`Failed to rename ${invalidDeviceID}: Permission Denied`
+		];
+
+		expect(stdout.split('\n')).to.include.members(log);
+		expect(stderr).to.equal('');
+		expect(exitCode).to.equal(1);
+	});
+
+	it('Fails to name a device owned by someone else', async () => {
+		const args = ['cloud', 'name', FOREIGN_DEVICE_ID, 'NOPE'];
+		const { stdout, stderr, exitCode } = await cli.run(args);
+		const log = [
+			`Renaming device ${FOREIGN_DEVICE_ID}`,
+			`Failed to rename ${FOREIGN_DEVICE_ID}: Permission Denied`
+		];
+
+		expect(stdout.split('\n')).to.include.members(log);
+		expect(stderr).to.equal('');
+		expect(exitCode).to.equal(1);
+	});
+
+	it('Starts a device signaling', async () => {
+		const args = ['cloud', 'nyan', DEVICE_NAME];
+		const { stdout, stderr, exitCode } = await cli.run(args);
+
+		expect(stdout).to.equal('');
+		expect(stderr).to.equal('');
+		expect(exitCode).to.equal(0);
+	});
+
+	it('Stops a device signaling', async () => {
+		const args = ['cloud', 'nyan', DEVICE_NAME, 'off'];
+		const { stdout, stderr, exitCode } = await cli.run(args);
+
+		expect(stdout).to.equal('');
+		expect(stderr).to.equal('');
+		expect(exitCode).to.equal(0);
+	});
+
+	// TODO (mirande): this currently includes all product devices..?
```

Comment:
You could delete this sub-command. It's not useful to signal all your devices.

Reply:
✅ in https://github.com/particle-iot/particle-cli/pull/565/commits/38013922fca97e2610271b3aaf419f0f36c3eb61